src/recipes/recipes.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { Component, useContext, useEffect, useState } from "react";
+import { RecipeModel } from "../data/recipe-model";
+import { Card, CardActionArea, CardContent, Typography, Theme, makeStyles, createStyles, Grid, Paper } from "@material-ui/core";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { RepositoriesContext } from "../context";
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            flexGrow: 1
+        },
+        card: {
+            width: 400,
+            padding: theme.spacing(2)
+        }
+    }));
+
+export const AllRecipes: React.FunctionComponent = props => {
+    const classes = useStyles(1);
+    const { recipeRepository } = React.useContext(RepositoriesContext);
+    const [recipes, setRecipes] = useState<RecipeModel[]>(null);
+
+    useEffect(() => {
+        const fetchRecipes = async () => {
+            const result = recipeRepository;
```

Comment:
This is very unclear - why are you storing the `recipeRepository` in `result`?

Why not just do `setRecipes(recipeRepository.getThreeRandomRecipes());`